helm/hpcc/templates/_helpers.tpl

```diff
@@ -313,6 +313,30 @@ Add Secret volume for a component
 {{- end -}}
 {{- end -}}
 
+{{/*
+Add sentinel-based probes for a component
+*/}}
+{{- define "hpcc.addSentinelProbes" -}}
+env:
+- name: "SENTINEL"
+  value: "{{ .name }}.sentinel"
+startupProbe:
+  exec:
+    command:
+    - cat
+    - "{{ .name }}.sentinel"
+  initialDelaySeconds: 5
```

Comment:
@richardkchapman this might be too high for some components - e.g. delaying how quickly the system can scale.  Is there a jira to improve the configuration?